src/components-examples/angular/form-field/form-field-group-vertical/form-field-group-vertical-example.ts

```diff
@@ -10,15 +10,14 @@ import { UntypedFormBuilder, UntypedFormGroup, Validators } from '@angular/forms
   templateUrl: 'form-field-group-vertical-example.html',
 })
 export class FormFieldGroupVerticalExample {
-  formGroup: UntypedFormGroup;
-  lastSubmission?: { name: string; date: Date };
+  formGroup = this._formBuilder.group({
+    name: ['', [Validators.required, Validators.minLength(3)]],
+    date: [new Date(), Validators.required],
+  });
 
-  constructor(formBuilder: UntypedFormBuilder) {
-    this.formGroup = formBuilder.group({
-      name: ['', [Validators.required, Validators.minLength(3)]],
-      date: [new Date(), Validators.required],
-    });
-  }
+  lastSubmission?: { name?: string | null; date?: Date | null };
```

Comment:
I am confused why here '| null' is necessary, as we do know, that the form doesn't contain a nullish form control